torch/_decomp/decompositions.py

```diff
@@ -4409,6 +4409,12 @@ def _weight_norm_interface(x, y, dim):
     norm = x.norm(2, keep_dim, keepdim=True)
     return x * (y / norm), norm
 
+@register_decomposition(aten.put)
```

Comment:
For the `test_has_decomposition` failure, you can run the test as
```
EXPECTTEST_ACCEPT=1 pytest test/test_decomp,py -k HasDecomp
```
and it will update the expected test output files
```
test/expect/HasDecompTest.test_has_decomposition.expect
test/expect/HasDecompTest.test_aten_core_opeartors.expect
```